test/powershell/engine/Formatting/BugFix.Tests.ps1

```diff
@@ -0,0 +1,34 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
+
+Describe 'Bug fixes related to formatting' -Tag CI {
+
+    It "Formatting for an object with no property/field should use 'ToString'" {
+        class Empty {
+            [String]ToString() { return 'MyString' }
+        }
+
+        $outstring = [Empty]::new() | Out-String
+        $outstring.Trim() | Should -BeExactly "MyString"
+    }
+
+    It "Formatting for an object with only hidden property should use 'ToString'" {
+        class Hidden {
+            hidden $Param = 'Foo'
+            [String]ToString() { return 'MyString' }
+        }
+
+        $outstring = [Hidden]::new() | Out-String
+        $outstring.Trim() | Should -BeExactly "MyString"
+    }
+
+    It 'Formatting for an object with no-hidden property should use the default view' {
+        class Params {
+            $Param = 'Foo'
+            [String]ToString() { return 'MyString' }
```

Comment:
What's the behaviour supposed to be when `ToString()` isn't overridden?

Reply:
Whether the `ToString()` method is present or not shouldn't affect how `Params` object is formatted by PowerShell in this case.

Reply:
Does it make sense in that case to have some test where `ToString()` isn't overridden as a control?

Reply:
Sure, I can add 3 more tests without the `ToString` override.